src/test/java/persistence/entity/EntityEntriesTest.java

```diff
@@ -0,0 +1,89 @@
+package persistence.entity;
+
+import domain.FixturePerson;
+import domain.Person;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.exception.PersistenceException;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+
+class EntityEntriesTest {
+    private EntityEntries entityEntries;
+    private Person person;
+
+    @BeforeEach
+    void setUp() {
+        entityEntries = new EntityEntries();
+        person = FixturePerson.create(null);
+    }
+
+    @Test
+    @DisplayName("EntityEntries 를 통해 EntityEntry 들을 person instance 를 이용해 저장하고 조회할 수 있다.")
+    void entityEntriesTest() {
+        entityEntries.addEntityEntry(person, Status.LOADING);
+
+        final Optional<EntityEntry> result = entityEntries.getEntityEntry(person);
+
+        assertSoftly(softly -> {
+            softly.assertThat(result).isNotEmpty();
+            softly.assertThat(result.get().getStatus()).isEqualTo(Status.LOADING);
+        });
+    }
+
+    @Test
+    @DisplayName("EntityEntries 를 통해 없는 person instance 조회시 Optional.empty 가 반환된다.")
+    void getEntityEntryNotExistTest() {
+        final Person newPerson = FixturePerson.create(null);
+        final Optional<EntityEntry> result = entityEntries.getEntityEntry(newPerson);
+
+        assertThat(result).isEqualTo(Optional.empty());
+    }
+
+    @Test
+    @DisplayName("updateEntityEntryStatus 를 통해 person instance 의 EntityEntry 의 상태를 변경할 수 있다.")
+    void updateEntityEntryStatusTest() {
+        entityEntries.addEntityEntry(person, Status.LOADING);
+
+        entityEntries.updateEntityEntryStatus(person, Status.MANAGED);
+        final Optional<EntityEntry> result = entityEntries.getEntityEntry(person);
+
+        assertSoftly(softly -> {
+            softly.assertThat(result).isNotEmpty();
+            softly.assertThat(result.get().getStatus()).isEqualTo(Status.MANAGED);
+        });
+    }
+
+    @Test
+    @DisplayName("updateEntityEntryStatus 를 통해 property 가 변한 person instance 의 EntityEntry 의 상태를 변경할 수 있다.")
+    void updateEntityEntryStatusWithChangedPropertyTest() {
+        entityEntries.addEntityEntry(person, Status.LOADING);
+
+        person.changeEmail("chagned@email.com");
+        entityEntries.updateEntityEntryStatus(person, Status.MANAGED);
+        final Optional<EntityEntry> result = entityEntries.getEntityEntry(person);
+
+        assertSoftly(softly -> {
+            softly.assertThat(result).isNotEmpty();
+            softly.assertThat(result.get().getStatus()).isEqualTo(Status.MANAGED);
+        });
```

Comment:
`객체에게 묻지말고 시켜라`, `객체간 비교하기` 이러한 관점을 고민해보셔도 좋을것 같아요.

--- 

저는 언어에서 어떠한 단어로 표현하는것이 중요한 부분이라고 생각해요.
언어의 표현방식으로도 우리가 바라보는 사고또한 변하지 않을까? 라는 생각을 어느정도 하고 있기때문이에요. 🤔 

우선은, 저는 테스트 코드에 하나의 `actual, expected`로 이름을 변경하고 다시 바라보니
결국 검증하고 싶었던것은 `"getEntityEntry" 가 내가 원하는 객체를 주는가?` 같아요.
그래서 아래와 같은 관점이 나오게 되었는데 참고 부탁드려요~

```suggestion
        entityEntries.addEntityEntry(person, Status.LOADING);
        person.changeEmail("chagned@email.com");
        entityEntries.updateEntityEntryStatus(person, Status.MANAGED);

        final EntityEntry actual = entityEntries.getEntityEntry(person).get();
        EntityEntry expected = new EntityEntry(Status.MANAGED);

        assertSoftly(softly -> {
            softly.assertThat(actual).isEqualTo(expected);
        });
```

Reply:
제가 여러부분이 부족하지만 가장 크게 느끼는 부분이 테스트코드입니다. 이러한 관점 넘 좋네요. 바로 반영하겠습니다